src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -179,65 +343,18 @@ private Task ExecuteSyntaxTreeActions(CancellationToken cancellationToken)
             out Compilation newCompilation,
             CancellationToken cancellationToken)
         {
-            options = options ?? AnalyzerOptions.Empty;
-            AnalyzerDriver analyzerDriver = compilation.AnalyzerForLanguage(analyzers, analyzerManager, cancellationToken);
-            newCompilation = compilation.WithEventQueue(analyzerDriver.CompilationEventQueue);
+            AnalyzerDriver analyzerDriver = compilation.AnalyzerForLanguage(analyzers, analyzerManager);
+            newCompilation = compilation.WithEventQueue(new AsyncQueue<CompilationEvent>());
```

Comment:
curious on perf on this. does it reuse cloned compilation if a compilation is already cloned?